src/python/pants/backend/codegen/ragel/subsystems/__init__.py

```diff
@@ -0,0 +1,6 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
```

Comment:
Should this file be empty? Ditto the other `__init__.py`

Reply:
D'oh! That's what I get for letting my IDE create files.